src/components/app.js

```diff
@@ -1,12 +1,31 @@
+import {
+  Link,
+  Route,
+  Routes,
+} from 'react-router-dom';
 import Calculator from './calculator';
 import QuoteDisplay from './fetchQuote';
+import Home from './home';
 
 function App() {
   return (
-    <div className="app">
-      <Calculator />
-      <QuoteDisplay />
-    </div>
+    <>
+      <header>
+        <nav className="navbar">
+          <div className="navbar-heading">Math Magicians</div>
+          <ul className="navbar-links">
+            <li><Link to="/">Home</Link></li>
+            <li><Link to="/calculator">Calculator</Link></li>
+            <li><Link to="/fetchQuote">Quotes</Link></li>
+          </ul>
+        </nav>
+      </header>
```

Comment:
- **[OPTIONAL]** This can be a component on its own, instead of adding this navbar here.